components/namespace-lister/base/deployment.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: namespace-lister
+  namespace: namespace-lister
+  labels:
+    apps: namespace-lister
+spec:
+  selector:
+    matchLabels:
+      apps: namespace-lister
+  replicas: 1
```

Comment:
please use 3 replicas and spread them across AZ.
You can see an example of how to do it in the deployment of nginx in the konflux-ui component.

Reply:
Fixed in 7c94977f.